src/ert/scheduler/scheduler.py

```diff
@@ -174,6 +191,34 @@ def add_dispatch_information_to_jobs_file(self) -> None:
         for job in self._jobs.values():
             self._update_jobs_json(job.iens, job.real.run_arg.runpath)
 
+    async def _monitor_and_handle_tasks(
+        self, scheduling_tasks: list[asyncio.Task[None]]
+    ) -> None:
+        pending: Iterable[asyncio.Task[None]] = (
+            list(self._tasks.values()) + scheduling_tasks
```

Comment:
We should rename self._tasks to something more specific, as it is "one-asyncio-task-per-job" we are talking about, not any asyncio task.

Reply:
self._job_tasks would probably do